AudioDeviceExample/README.md

```diff
@@ -8,11 +8,16 @@ The example demonstrates the following custom audio device(s):
 
 Uses a RemoteIO audio unit to playback stereo audio at up to 48 kHz. In contrast to `TVIDefaultAudioDevice`, this class does not record audio and is intended for high quality playback. Since recording is not needed this device does not use the built in echo cancellation provided by CoreAudio's VoiceProcessingIO audio unit nor does it require microphone permissions from the user.
 
-**ExampleAudioEngineDevice**
+**ExampleAVAudioEngineDevice**
 
-Coming soon.
+Uses the CoreAudio's VoiceProcessingIO audio unit to record and playback at up to 48KHz. Since this example supports audio capturing, the audio device uses the built-in echo cancellation provided by CoreAudio's VoiceProcessingIO audio unit. The example uses AVAudioEngine in manual rendering mode for mixing the Remote Participant's audio and audio from a file. CoreAudio receives the mixed audio samples from the AVAudioUnit's 
+output node.
```

Comment:
`Uses CoreAudio's`.

I think the first two sentences could be condensed down to:

`Uses CoreAudio's VoiceProcessingIO audio unit to playback and record audio at up to 48KHz with echo cancellation.`